cmd/pj-rehearse/main.go

```diff
@@ -148,6 +148,10 @@ func main() {
 	}
 
 	changedPresubmits := diffs.GetChangedPresubmits(prowConfig, prowPRConfig, logger)
+	if len(changedPresubmits) == 0 {
+		logger.Info("no jobs to rehearse have been found")
+		gracefulExit(o.noFail, "")
```

Comment:
This would error-out on no rehearsed jobs, we probably do not want that.

Reply:
addressed